README.md

```diff
@@ -1,47 +1,172 @@
 # Knowledge sharing toolkit
 
-**NOTE: THIS IS NOT YET USABLE!!!** It should become completely deployable by
-2016-02-26.
+[![Build Status](https://travis-ci.org/18F/knowledge-sharing-toolkit.svg?branch=master)](https://travis-ci.org/18F/knowledge-sharing-toolkit)
+
+**NOTE: THIS IS _ALMOST_, BUT NOT YET USABLE!!!** It will be completely ready
+by 2016-03-04 (or sooner).
 
 The knowledge sharing toolkit contains the [Hub](https://github.com/18F/hub/),
 [18F Pages](https://github.com/18F/pages-server), and [Team
 API](https://github.com/18F/team-api-server/). These are lightweight services
 that enable a team to collect and radiate institutional knowledge and
-information. This project contains Docker components for these services, to
-enable rapid deployment of the entire suite.
-
-## Dependencies
-- Ruby >= 2.3.0
-- Docker
+information. This project contains [Docker](https://www.docker.com/)
+components for these services, to enable rapid deployment of the entire suite.
 
 ## Installation
 
-To install all the images at once: `./go build_images`
+These steps are necessary on your development machine. They are included in
+the [Deployment](#deployment) section below as well.
 
-To install specific a specific image or specific [images](#images)
+1. Install [Ruby](https://ruby-lang.org/) on your system. The [`./go`
+   script](./go) command line interface requires version 2.3.0 or greater. You
+   may wish to first install a version manager such as
+   [rbenv](https://github.com/rbenv/rbenv) to manage and install different
+   Ruby versions.
 
-```
+1. Install [Docker](https://www.docker.com/) on your system. The commands
+   encapsulated in the [`./go` script](./go) are based on version 1.10.0.
+
+1. Run `docker-machine start` to start the Docker host, followed by
+   `eval $(docker-machine env)` to configure your shell environment.
+
+1. After cloning this repository, install all the images by running `./go
+   build_images` within your copy of the repository.
+
+## Deployment
+
+1. Install your public SSH key on the remote host machine.
+
+1. Set the `REMOTE_HOST` and `REMOTE_ROOT` variables in the `./go` script as
+   necessary.
+
+1. Run `./go init_remote` if running on a brand new server. Otherwise run
```

Comment:
Well, it's a melting snowflake, I hope. By making it easier to bring the system up on a new machine via the `./go` script, running containerized services, I hope the next step is finding better ways to provision it.